formatter-codequality/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "markdownlint-cli2-formatter-codequality",
+  "version": "0.0.1",
+  "description": "An output formatter for markdownlint-cli2 that writes results to a JSON file following the GitLab code quality syntax",
```

Comment:
Please make this the same as the initial blockquote in README (okay to leave out the parenthesized part at the end).